l10n_es_aeat_sii_oca/views/aeat_sii_map_view.xml

```diff
@@ -17,14 +17,20 @@
         <field name="model">aeat.sii.map</field>
         <field name="arch" type="xml">
             <form string="Aeat SII Map">
-                <group col="4">
-                    <field name="name" />
-                    <field name="date_from" />
-                    <field name="date_to" />
-                </group>
-                <group string="Mapping Lines" col="4">
-                    <field name='map_lines' nolabel="1" />
-                </group>
+                <sheet>
+                    <group>
```

Comment:
No es mejor dejar los dos campos date en un mismo grupo

```
<group>
    <field name="name"/>
</group>
<group>
    <field name="date_from"/>
    <field name="date_to"/>
</group>
```

Sin mi propuesta:
![Selección_395](https://github.com/OCA/l10n-spain/assets/6359121/e3d94f6c-881a-459d-8d52-3d84df43dfee)

Con mi propuesta:
![Selección_396](https://github.com/OCA/l10n-spain/assets/6359121/5bb7ade5-b982-49d7-9fb5-1a0c82880ac9)

Un saludo

Reply:
Done, Thanks for the review!